### ISSUE TITLE
Using postcss-nesting with 2.0.1 -- text-[color] issues?

### ISSUE THREAD

**Issue by brycewray** [tailwindlabs/tailwindcss#2800]
Am wondering if anybody else is seeing the following problem with TWCSS 2.0.1. I am using the **postcss-nesting** plugin as suggested in the documentation; and, in **nested** CSS which includes `text-[color]` statements (for example, `text-white` or `text-blue-200`), I am getting an error message like the following (with my local directory info obscured):

```
CssSyntaxError: /Users/[MyLocal]/[MyRepo]/src/assets/css/nav.css:9:33: The `text-white` class does not exist. If you're sure that `text-white` exists, make sure that any `@import` statements are being properly processed before Tailwind CSS sees your CSS, as `@apply` can only be used for classes in the same CSS tree.
```

The following contains the CSS to which it objects — specifically, in the nested `& a` within `.nav__item`:
```
.nav__item {
  @apply font-sans p-0 m-0 text-blue-100;

  & a {
    @apply block py-1 px-4 mb-1 text-white no-underline transition-opacity duration-500 ease-linear;
    }
}
```

I presume it has to do with the nesting, because the `text-blue-100` in the earlier line causes no problem. **Note**: The README for the **postcss-nesting** plugin says in part, "deprecating nested at-rules has been temporarily delayed"; although it's referring to `@media` statements specifically, would this also affect TWCSS `@apply` statements as well? If so, it would seem to preclude using **postcss-nesting** with TWCSS 2.0.1. (Before 2.0.1, I had used **postcss-preset-env** to handle this and other items but, when I began running into this issue today, I stopped using it in favor of separate installations of **autoprefixer** and **postcss-nesting**, since the TWCSS 2.x documentation does indicate a requirement for those. **However**, I then noticed the documentation also mentioned the option of using **postcss-preset-env** instead, so I reverted to that rather than separate **autoprefixer** and **postcss-nesting**, but still saw the same anomalous behavior described above.)

Many thanks in advance for any feedback on whether anyone else is experiencing this issue. Repo link (branch): https://github.com/brycewray/eleventy_solo/tree/twcss-v2

**Comment by kuroi93** [comment]
With the new Tailwind, you have to declare the colors you want to use in the configuration file as shown in the documentation :

[Link to the #color section of the documentation](https://tailwindcss.com/docs/customizing-colors#color-palette-reference)

If you want to use all of the colors, you can write something like that :
``` js
module.exports = {
  theme: {
    colors: {
      white: '#FFFFFF',
      black: '#000000',
      bluegray: colors.blueGray,
      coolgray: colors.coolGray,
      gray: colors.gray,
      truegray: colors.trueGray,
      warmgray: colors.warmGray,
      red: colors.red,
      orange: colors.orange,
      amber: colors.amber,
      yellow: colors.yellow,
      lime: colors.lime,
      green: colors.green,
      emerald: colors.emerald,
      teal: colors.teal,
      cyan: colors.cyan,
      lightblue: colors.lightBlue,
      blue: colors.blue,
      indigo: colors.indigo,
      violet: colors.violet,
      purple: colors.purple,
      pink: colors.pink,
      rose: colors.rose
    },
  },
}

**Comment by brycewray** [comment]
@kuroi93 Thanks; very helpful. Wondering, however, what happened to `text-transparent`, which I had been using in TWCSS 1.x to override underlining of links in favor of `border-bottom` treatment instead. Searching the 2.x docs for "transparent" isn't getting anything, although that may be more of an Algolia issue than an actual lack of documentation thereof.

**Comment by brycewray** [comment]
Thanks, @kuroi93. Closing this.

**Comment by rwenz3l** [comment]
I usually only use the `https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css` file as a drop-in solution for small sites.. do I need to build that myself now or is there anyway I can get my colors back? I'm pretty sure `teal` was available in 2.0.1 but it's gone in 2.0.2?
